### PR TITLE
Add tests for aggregator functions (meant for AttributeTables, but stored in Attribute classes)

### PR DESCRIPTION
**Pull request by bharathcs**

### CODE REVIEW

File: src/test/java/donnafin/model/person/LiabilityTest.java [hunk 1]
```diff
@@ -83,4 +86,37 @@ public void variable_commissionToString_valid() throws MoneyException {
...
+        // EP: 1 Liability
+        assertEquals(
+                "Total Liability Value: $2000.00",
+                Liability.TABLE_CONFIG.aggregatorLabelCreator.applyOn(List.of(VALID_LIABILITY))
+        );
+
+        // EP: 0 Liability
+        assertEquals("", Liability.TABLE_CONFIG.aggregatorLabelCreator.applyOn(List.of()));
+
+        // EP: Commissions exceeds Long.MAX_VALUE
```
Comment (sheehui) [inline src/test/java/donnafin/model/person/LiabilityTest.java:111]: commissions :(

File: src/test/java/donnafin/model/person/AssetTest.java [hunk 2]
```diff
@@ -85,4 +88,35 @@ public void variable_commissionToString_valid() throws MoneyException {
...
+        // EP: 1 Asset
+        assertEquals(
+                "Total Asset Value: $2000.00",
+                Asset.TABLE_CONFIG.aggregatorLabelCreator.applyOn(List.of(VALID_ASSET))
+        );
+
+        // EP: 0 Asset
+        assertEquals("", Asset.TABLE_CONFIG.aggregatorLabelCreator.applyOn(List.of()));
+
+        // EP: Commissions exceeds Long.MAX_VALUE
```
Comment (sheehui) [inline src/test/java/donnafin/model/person/AssetTest.java:111]: commissions :(

**Review by sheehui — COMMENTED** [review]
GitHub copilot hehe 👀

File: src/test/java/donnafin/model/person/LiabilityTest.java (hunk 1 above)
Comment (bharathcs) [reply src/test/java/donnafin/model/person/LiabilityTest.java:111]: # 👨‍✈️🤦

Comment (bharathcs) [reply src/test/java/donnafin/model/person/LiabilityTest.java]: Resolved by bdc65639

File: src/test/java/donnafin/model/person/AssetTest.java (hunk 2 above)
Comment (bharathcs) [reply src/test/java/donnafin/model/person/AssetTest.java]: Resolved by bdc65639

**Comment by bharathcs** [comment]
Nice catch! 🤦

**Review by sheehui — APPROVED** [review]
yay we're back at 70% code coverage 🎉

**Pull request merged**